test/per_file_coverage.sh

```diff
@@ -3,34 +3,33 @@
 # directory:coverage_percent
 # for existing directories with low coverage.
 declare -a KNOWN_LOW_COVERAGE=(
-"source/common/api:72.9"
-"source/common/api/posix:71.8"
-"source/common/common:96.1"
+"source/common/api:75.4"
+"source/common/api/posix:74.0"
+"source/common/common:96.3"
 "source/common/common/posix:94.1"
 "source/common/crypto:0.0"
-"source/common/event:92.9" # Emulated edge events guards don't report LCOV
-"source/common/filter:96.3"
-"source/common/filter/http:96.3"
+"source/common/event:93.4" # Emulated edge events guards don't report LCOV
+"source/common/filter:96.5"
+"source/common/filter/http:96.5"
 "source/common/http/http3:50.0"
 "source/common/json:90.6"
-"source/common/network:95.1"
+"source/common/network:95.2"
 "source/common/protobuf:94.6"
 "source/common/signal:84.5" # Death tests don't report LCOV
 "source/common/singleton:95.1"
 "source/common/thread:0.0" # Death tests don't report LCOV
-"source/common/matcher:92.8"
+"source/common/matcher:93.3"
 "source/common/tracing:94.9"
 "source/common/watchdog:42.9" # Death tests don't report LCOV
 "source/exe:93.8"
-"source/extensions:96.3"
 "source/extensions/common/crypto:91.5"
 "source/extensions/common/tap:95.9"
-"source/extensions/common/wasm:95.3"
+"source/extensions/common/wasm:95.4"
```

Comment:
I adjusted this one down to 95.3 a while back because coverage of some lines is not fully deterministic.  Would it be possible to verify this won't make CI coverage checks flaky?